### ISSUE TITLE
Sharing dialog exposes user id on current master

### ISSUE THREAD

**Issue by blizzz** [nextcloud/server#12164]
This one is not supposed to be shown to end users:

![sidebar](https://user-images.githubusercontent.com/2184312/47784640-78247e80-dd06-11e8-8b05-e90565bb13dc.png)

Also, the tabs are aligned weirdly @nextcloud/designers @nextcloud/sharing

**Comment by jancborchardt** [comment]
cc @juliushaertl for the consolidation. Yeah, these cryptic LDAP IDs (right?) should indeed not show. Is there nothing else human-readable to show there?

@blizzz about the sidenote regarding the tabs, this is fixed in https://github.com/nextcloud/server/pull/12180, please review. :)

**Comment by blizzz** [comment]
> cc @juliushaertl for the consolidation. Yeah, these cryptic LDAP IDs (right?) should indeed not show. Is there nothing else human-readable to show there?

Yes, LDAP in this case, but it could be any arbitrary backend. The display name is already being shown.
